src/hooks/context/contextService.spec.ts

```diff
@@ -55,6 +55,7 @@ describe('ContextService.autoDetect', () => {
 	const contextService = new ContextService();
 
 	test("Should return undefined when does not detects anyfile", () => {
+		testing.deleteSpecFileAtWorkingDir();
```

Comment:
Please also add test for `createSpecFileAtWorkingDir` function

Reply:
Should I add complete tests for `ContextTestingHelper`, I didn't write it before because I created it as a helper class for creating scenarios for tests.

Reply:
Ok, sorry, I thought that new functions are from normal codebase 🤦🏼  So you need to add tests.